cypress/support/step_definitions/dashboard/list_settings/i_enter_{string}_in_the_email_input.js

```diff
@@ -0,0 +1,5 @@
+/* eslint-disable */
```

Comment:
what's the warning/error causing this?

Reply:
Might be just a warning that I'm getting but it doesn't affect the code, it's just something in VSCode.

<img width="1149" alt="Screenshot 2022-10-19 at 15 30 38" src="https://user-images.githubusercontent.com/1377253/196720499-5ef7ca64-bbc8-49d0-a665-b8a8463d54b4.png">